docs/api-reference/reference.asciidoc

```diff
@@ -2104,6 +2106,27 @@ ZyncConfig configures app behavior for Zync
 |===
 
 
+[id="{anchor_prefix}-github-com-3scale-saas-operator-api-v1alpha1-zyncrailsconsolespec"]
+==== ZyncRailsConsoleSpec 
+
+ZyncRailsConsoleSpec configures the App component of System
```

Comment:
wrong comment?

Reply:
fixed in https://github.com/3scale-ops/saas-operator/pull/263/commits/f7909daed91e94d2e466ff1371f807cad17fc5d2